### PR TITLE
Feature/climber5.0

### PR DESCRIPTION
**Pull request by bookslifestyle**
PTAL

### CODE REVIEW

**Comment by duane701** [comment]
Need to rebase to the most recent master in order to merge this PR.

**Comment by duane701** [comment]
Reviewing overall instead of per-commit.

File: src/main/cpp/RobotContainer.cpp [hunk 1]
```diff
@@ -24,13 +28,28 @@ RobotContainer::RobotContainer()
...
+  mTelescope.SetDefaultCommand
+  (
+    TelescopeRise
+    (
+      mTelescope,
+      [this] { return codriver.GetY(JoystickHand::kLeftHand); }
+    )
+  );
+
+  frc::SmartDashboard::PutData("Telescope Rise", new TelescopeRise(mTelescope, [this] {return 0.1;}));
```
Comment (duane701) [inline src/main/cpp/RobotContainer.cpp:40]: Do we need these if we have the joystick?

Comment (duane701) [reply src/main/cpp/RobotContainer.cpp:40]: Also these values might be a little low.

**Review by duane701 — CHANGES_REQUESTED** [review]
LGTM

One comment that doesn't need a change.

You do need to rebase however.

**Comment by bookslifestyle** [comment]
replaced by !e3c7321bbb3b8208306801e60eb566a0c036028b

**Pull request closed without merging**